content/en/real_user_monitoring/session_replay/mobile/setup_and_configuration.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Mobile Session Replay Setup and Configuration
+kind: documentation
+description: Setting up and configuring Mobile Session Replay.
+aliases:
+further_reading:
+    - link: '/real_user_monitoring/session_replay/mobile'
+      tag: Documentation
+      text: Mobile Session Replay
+    - link: '/real_user_monitoring/session_replay/mobile/app_performance'
+      tag: Documentation
+      text: How Mobile Session Replay Impacts App Performance
+    - link: '/real_user_monitoring/session_replay/mobile/privacy_options'
+      tag: Documentation
+      text: Mobile Session Replay Privacy Options
+    - link: '/real_user_monitoring/session_replay/mobile/troubleshooting'
+      tag: Documentation
+      text: Troubleshoot Mobile Session Replay
+    - link: '/real_user_monitoring/session_replay'
+      tag: Documentation
+      text: Session Replay
+---
+
+## Setup
+
+{{< tabs >}}
+{{% tab "Android" %}}
+
+All Session Replay SDK versions can be found in the [Maven Snapshots Repository][1].
+
+To set up Mobile Session Replay for Android:
+
+1. Make sure you've [setup and initialized the Datadog Android RUM SDK][2] with views instrumentation enabled.
+
+2. Once the Datadog SDK and Session Replay SDK dependencies are imported, you can enable the feature when configuring the SDK:
+   ```kotlin
+      val config = Configuration.Builder(
+         logsEnabled = true,
```

Comment:
Like I mentioned in previous comment above, this part should not longer be here as we already mention it at point 1 that we should first enable RUM.